### PR TITLE
Send the subscription key to the Community push proxy

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

File: lib/Push.php [hunk 1]
```diff
@@ -379,11 +379,20 @@ protected function sendNotificationsToProxies(): void {
...
 			try {
-				$response = $client->post($proxyServer . '/notifications', [
+				$requestData = [
 					'body' => [
 						'notifications' => $notifications,
 					],
-				]);
+				];
+
+				if ($proxyServer === 'https://push-notifications.nextcloud.com') {
```
Comment (PVince81) [inline lib/Push.php:388]: should we rtrim slashes on `$proxyServer` just in case ?

**Review by PVince81 — APPROVED** [review]
:+1: see minor comment if relevant

File: lib/Push.php (hunk 1 above)
Comment (nickvergessen) [reply lib/Push.php:388]: Already trimmed here: https://github.com/nextcloud/notifications/blob/49fb541f2729994c0871b62b47143cfd0a4d8201/lib/Push.php#L347

**Pull request merged**